admin_manual/installation/nginx-root.conf.sample

```diff
@@ -171,7 +171,12 @@ server {
         fastcgi_pass php-handler;
 
         fastcgi_intercept_errors on;
-        fastcgi_request_buffering off;
+		fastcgi_request_buffering off;
+
+		# Uncomment the following lines to avoid 504 errors for large files (PHP-FPM).
+		# fastcgi_read_timeout 1800s;
+		# fastcgi_send_timeout 1800s;
+		# fastcgi_connect_timeout 1800s;
```

Comment:
```suggestion
        fastcgi_request_buffering off;

        # Uncomment the following lines to avoid 504 errors for large files (PHP-FPM).
        #fastcgi_read_timeout 60s;
        #fastcgi_send_timeout 60s;
        #fastcgi_connect_timeout 60s;
```